src/store/index.ts

```diff
@@ -0,0 +1,4 @@
+import createStore from './createStore';
+
+export * from './mode';
+export { createStore };
```

Comment:
I believe `export { default as createStore } from './createStore';` should work here.